diff -rupN udev-182.orig/rules/42-usb-hid-pm.rules udev-182/rules/42-usb-hid-pm.rules
--- udev-182.orig/rules/42-usb-hid-pm.rules	2015-05-07 15:40:58.818986964 -0400
+++ udev-182/rules/42-usb-hid-pm.rules	2015-05-07 15:47:00.386986921 -0400
@@ -49,6 +49,7 @@ ACTION=="add", SUBSYSTEM=="usb", ATTR{id
 # And skip it for devices which are external but say "fixed"
 ACTION=="add", SUBSYSTEM=="usb", SUBSYSTEMS=="usb", ATTRS{removable}=="removable", GOTO="usb_hid_pm_end"
 ACTION=="add", SUBSYSTEM=="usb", SUBSYSTEMS=="usb", ATTRS{removable}=="unknown", GOTO="usb_hid_pm_end"
-ACTION=="add", SUBSYSTEM=="usb", ATTR{bInterfaceClass}=="03", ATTRS{removable}=="fixed", TEST=="../power/control", ATTR{../power/control}="auto"
++# XC-9900 Comment that line as it is messing up with USB2 HID devices
++#ACTION=="add", SUBSYSTEM=="usb", ATTR{bInterfaceClass}=="03", ATTRS{removable}=="fixed", TEST=="../power/control", ATTR{../power/control}="auto"
 LABEL="usb_hid_pm_end"
 
